src/SFML/Graphics/Font.cpp

```diff
@@ -752,23 +752,28 @@ bool Font::setCurrentSize(unsigned int characterSize) const
 
 
 ////////////////////////////////////////////////////////////
-Font::Page::Page(bool smooth)
+Font::Page::Page(bool smooth) :
+texture(
+    [smooth]()
+    {
+        // Make sure that the texture is initialized by default
+        Image image({128, 128}, Color::Transparent);
+
+        // Reserve a 2x2 white square for texturing underlines
+        for (unsigned int x = 0; x < 2; ++x)
+            for (unsigned int y = 0; y < 2; ++y)
+                image.setPixel({x, y}, Color::White);
+
+        // Create the texture
+        auto maybeTexture = sf::Texture::loadFromImage(image);
+        if (!maybeTexture)
+            err() << "Failed to load font page texture" << std::endl;
+
+        auto newTexture = maybeTexture.value();
+        newTexture.setSmooth(smooth);
+        return newTexture;
```

Comment:
I kept this in my latest revision of this PR. I don't like what I wrote but can't picture anything better at the moment.

Reply:
The natural next step would be that Font::Page is created with a factory function.

Reply:
I went ahead and made this change because it seems sensible. What happened is now the null check occurs downstream in `sf::Font::loadPage` where the pages are actually constructed. If construction fails inside `loadPage` then what do we do? I chose to `assert` that loading succeeds but we could just as well use `.value()` in which case we're back to the same solution as what I previously wrote. 

Should we go so far as to make `sf::Font::loadPage` return something nullable and put the onus users? That's possible but I don't love the idea.